lib/cdo/graphics/certificate_image.rb

```diff
@@ -24,18 +28,22 @@ def create_certificate_image2(image_path, name, params={})
   background
 end
 
-def create_course_certificate_image(name, course=nil, sponsor=nil)
+def create_course_certificate_image(name, course=nil, sponsor=nil, course_title=nil)
   name = name.gsub(/@/,'\@')
   name = ' ' if name.empty?
 
-  if course == '20hours' || course == 'hoc' || course == '20-hour' || course == 'hourofcode' || course == 'starwars'
-    # only need to fill in name
-    image_file = (course == '20hours' || course == '20-hour') ? '20hours_certificate.jpg' : 'hour_of_code_certificate.jpg'
-    vertical_offset = course == '20hours' || course == '20-hour' ? -115 : -110
-    image = create_certificate_image2(pegasus_dir('sites.v3', 'code.org', 'public', 'images', image_file), name, y: vertical_offset)
+  course ||= ScriptConstants::HOC_NAME
```

Comment:
Not sure, but it's happened 11 times (Brendan and I were guessing that someone was messing with hidden inputs in a form)

> On Dec 1, 2015, at 10:05, Mehal Shah <notifications@github.com> wrote:
> 
> In lib/cdo/graphics/certificate_image.rb:
> 
> >    name = name.gsub(/@/,'\@')
> >    name = ' ' if name.empty?
> >  
> > -  if course == '20hours' || course == 'hoc' || course == '20-hour' || course == 'hourofcode' || course == 'starwars'
> > -    # only need to fill in name
> > -    image_file = (course == '20hours' || course == '20-hour') ? '20hours_certificate.jpg' : 'hour_of_code_certificate.jpg'
> > -    vertical_offset = course == '20hours' || course == '20-hour' ? -115 : -110
> > -    image = create_certificate_image2(pegasus_dir('sites.v3', 'code.org', 'public', 'images', image_file), name, y: vertical_offset)
> > +  course ||= ScriptConstants::HOC_NAME
> Why would course ever be nil?
> 
> —
> Reply to this email directly or view it on GitHub.
>